dbms/src/Flash/tests/gtest_planner_interpreter.cpp

```diff
@@ -105,15 +105,12 @@ try
 Union: <for test>
```

Comment:
Because of https://github.com/pingcap/tiflash/pull/5143/files#diff-d082f6e1f39c9f0441591de9ad75163aca6150f76c356eba4de577c65ae676d2R109-R122, the useless Expression: <before projection> and Expression: <before aggregating> has been removed.